.github/workflows/coding-standards.yml

```diff
@@ -1,14 +1,10 @@
-name: "Coding Standards"
+name: "CS"
 
 on:
   pull_request:
-    branches:
-      - "*.x"
-      - "master"
   push:
-    branches:
-      - "*.x"
-      - "master"
```

Comment:
How can we having double the jobs when making a PR from origin to origin?

Reply:
that is impossible as far as I know

Reply:
> normally

Yes, normally… keyword… sometimes some maintainers will push to origin, deliberately or not.

Reply:
if "not normally", then I am aware GH displays both results in PR checks, but that is fine, it does not affect the code in any way, it is purerly how GH display results 8-)

Reply:
I don't know, @morozov what do you think?

Reply:
that is IMO correct, as once with your direct (source branch) commit and once with (simulated) merge commit (into target/PR branch)

Reply:
> It's not just display IMO, the tests are really run twice.

That's the point. This is the reason why I wanted to introduce these filters.

Reply:
I know, but what is the real issue with it?

When I was implementing GH actions for other repos, I noticed this too, but I think that it is fully fine when CI tests are run everywhere, also, here is my research on top php repos:
- https://github.com/symfony/symfony/blob/5.x/.github/workflows/tests.yml
- https://github.com/sebastianbergmann/phpunit/blob/master/.github/workflows/ci.yml
- https://github.com/composer/composer/blob/master/.github/workflows/continuous-integration.yml
- https://github.com/laravel/framework/blob/8.x/.github/workflows/tests.yml

all of them trigger CI on push event

Reply:
ok to merge?